uiowa/uiowa
# Commencement updates

Resolve #7589 

# How to test

- Code review
```
blt ds --site commencement.uiowa.edu && ddev drush @commencement.local commencement-events && ddev drush @commencement.local uli node/add/page
```
- Add a few sessions (including program).
- Confirm field is now named Session (Spring, Fall, etc.)
- https://commencement.uiowa.ddev.site/node/add/page
- Add a page, save, layout, add block, Program Archive.
- Confirm you see year and session columns and that they are both sortable. Default sort should be by year, descending.
- Review imported events and confirm they are unpublished.
- Inspect HTML on event and ceremonies to see that the event and place metatag schema info is being displayed on events and venues.
- Note any unexpected behavior or possible regressions.
